modules/http4s/src/main/scala/com/evolutiongaming/smetrics/MetricsRoutes.scala

```diff
@@ -0,0 +1,16 @@
+package com.evolutiongaming.smetrics
+
+import cats.effect.Sync
+import org.http4s.{HttpRoutes, Response, Status}
+
+object MetricsRoutes {
```

Comment:
are you sure this should be part of this lib?

Reply:
I'm not a maintainer of this lib, so feel free to correct my thoughts if I'm missing something. My logic is next: we need metrics routes in several our project. I assume other teams if they are using http4s and metrics for it, may want to also expose these routes, since they already definitely have http4s server. And this routes do not take a lot of code or dependencies. So we gain convenience for teams that use http4s and don't loose anything.

Reply:
Could you ask in your team for additional opinion.
As I think that this code has not useful reusable logic, and hence could be just copied across projects. On contrary it has `case GET -> Root / "metrics" =>` hardcoded the way metrics are exposes 

That's not a strong opinion from my side :)